### ISSUE TITLE
Still can't get remaining time to be -:-- when state is Off/Standby/-

### ISSUE THREAD

**Issue by FS1961** [phrz/lg-washer-dryer-card#26]
@phrz - I've reset the Wifi and now the card is much closer to correct and updates at regular intervals - Thanks!

However, the display is still not working. Please see screen shots below:

<img width="497" alt="Screenshot 2023-02-18 at 9 51 18 AM" src="https://user-images.githubusercontent.com/98458806/219851608-bdceddb5-6c29-4601-9990-f4a12c3d4a50.png">
<img width="327" alt="Screenshot 2023-02-18 at 9 53 01 AM" src="https://user-images.githubusercontent.com/98458806/219851611-175ab3b0-1f0f-41bf-b8b1-3c1fe402f7f7.png">

I've made the changes to the sensor in config.yml

  - platform: template
    sensors:
      washer_door_lock:
        friendly_name: "Washer Door Lock"
        value_template: "{{ state_attr('sensor.front_load_washer','door_lock') }}"

      washer_time_display:
       friendly_name: "Washer Time Display"
       value_template: >
        {% if is_state('sensor.front_load_washer_run_state', '-') %}
        {% elif is_state('sensor.front_load_washer_run_state', 'Off') %}
        {% elif is_state('sensor.front_load_washer_run_state', 'Standby') %}
          -:--
        {% else %}
          {{ state_attr("sensor.front_load_washer","remain_time").split(":")[:-1] | join(':') }}   
        {% endif %}

Still no joy! :(

**Comment by phrz** [comment]
Have you rebooted Home Assistant? The config you've shown should blank out the time display based on the state you currently have.

**Comment by FS1961** [comment]
Yes - I’ve actually rebooted the host as well and not just restarted the HA. it’s weird.

**Comment by phrz** [comment]
Can you confirm that your Run State entity is actually named `sensor.front_load_washer_run_state`? If it isn't, I can see why the code would fall back to displaying the time.

**Comment by FS1961** [comment]
<img width="570" alt="Screenshot 2023-02-19 at 6 41 16 AM" src="https://user-images.githubusercontent.com/98458806/219929613-8692198d-326e-4a61-964e-27faefc7b010.png">

**Comment by FS1961** [comment]
Okay, I think I found it - in the card itself! Somewhere I guess I edited the card incorrectly when making the initial changes originally. Sorry to have been such a bother - but on the bright side, I've learned a little!

I changed this:

- type: state-label
        entity: sensor.front_load_washer_remaining_time

**_to_**: 

- type: state-label
        entity: sensor.washer_time_display
<img width="493" alt="Screenshot 2023-02-19 at 7 35 48 AM" src="https://user-images.githubusercontent.com/98458806/219932983-4ac16aef-41ac-4f61-8c7f-03d2c9a804a4.png">
